.github/workflows/unit-tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install robotframework${{ matrix.rf-version }}
+        pip install -r /tests/packages/{{ matrix.rf-version }}/requirements.txt
```

Comment:
Error from checks: `Hint: It looks like a path. File '}}/requirements.txt' does not exist.`. Most likely missing `$` before `{{`

Reply:
done